Test failure: ARM assembly vetting fails on Remainder

Running Go test on my machine:
```
--- FAIL: TestVetMath32 (3.45s)
    --- FAIL: TestVetMath32/GOOS=linux_GOARCH=arm (0.33s)
        math32_test.go:55: # github.com/chewxy/math32.test
            github.com/chewxy/math32.archRemainder: nosplit stack over 800 byte limit
            github.com/chewxy/math32.archRemainder<0>
                grows 0 bytes, calls github.com/chewxy/math32.archRemainder<0>
                infinite cycle
             exit status 1
FAIL
FAIL    github.com/chewxy/math32        3.449s
FAIL
```

--- a/stubs_arm.s
+++ b/stubs_arm.s
@@ -14,7 +14,7 @@ TEXT ·archLog(SB),NOSPLIT,$0
 
 // func archRemainder(x, y float32) float32
 TEXT ·archRemainder(SB),NOSPLIT,$0
-	B ·archRemainder(SB)
+	B ·remainder(SB)
 
 // func archSqrt(x float32) float32
 TEXT ·archSqrt(SB),NOSPLIT,$0

--- a/stubs_arm64.s
+++ b/stubs_arm64.s
@@ -4,6 +4,6 @@
 TEXT ·archLog(SB),NOSPLIT,$0
 	B ·log(SB)
 
-// func archRemainder(x float32) float32 // TODO
+// func archRemainder(x, y float32) float32 // TODO
 // TEXT ·archRemainderTODO(SB),NOSPLIT,$0
 //	B ·remainder(SB)

